exporter/googlecloudexporter/internal/resourcemapping/custommapping.go

```diff
@@ -14,14 +14,30 @@ var (
 	monitoredResourceLabelPrefix = "gcp."
 )
 
-// CustomMonitoredResourceMapping allows mapping from OTel resources to
+// CustomLoggingMonitoredResourceMapping allows mapping from OTel resources to
+// Monitored Resources defined here:
+// https://cloud.google.com/monitoring/api/resources
+//
+// The monitored resource type is extracted from the `gcp.resource_type`
+// attribute. And the monitored resource labels are extracted from resource
+// attributes with the prefix `gcp.type.`.
```

Comment:
This line of the description should probably match the one below for `CustomMetricMonitoredResourceMapping`, since I assume `gcp.<monitored resource type>` is how this works as well based on my read of the code.

Reply:
Fixed in: [74c436a](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40005/commits/74c436a2e9b68ab59d601b5c9e2b1c2c0b464c73)